changelog/pending/20250304--sdk-python--trying-to-convert-outputs-to-strings-now-logs-a-warning.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: sdk/python
+  description: Trying to convert outputs to strings now logs a warning
```

```suggestion
  description: Log a warning when trying to convert outputs to strings now logs a warning
```

Same comment here as in the dotnet PR :)